packages/react-email/package.json

```diff
@@ -6,7 +6,7 @@
     "email": "./cli/index.js"
   },
   "scripts": {
-    "build": "tsup",
+    "build": "tsup ",
```

Comment:
why is that?

Reply:
Oh, that's an artifact from when I was testing out the demo deploy to Vercel. I was just adding an extra space at the end and removing for it to trigger again